hrtime_migrate/migrate_sqlite.py

```diff
@@ -0,0 +1,170 @@
+import sqlite3, os, json, sys
+from sqlite3 import Error
+
+#json_path = ["db_card.json", "db_registration.json", "db_user.json"]
+
+database = "hrtime.db"
```

Comment:
Please place all the configuration to Mafike environemtn varialbes. 
Makefile :
```bash
HRTIME_DB_FILE="hritme.db"

data\hrtime\init:
	python .\hrtime_migrate\migrate_sqlite.py recreate
```
and your python file add something like that
```python
os.environ['HRTIME_DB_FILE']
```